org.jfree.data.jdbc.JDBCXYDataset.close : r1 = $r3 >>>>>>>> org.jfree.data.jdbc.JDBCXYDataset.close : $r4 = <java.lang.System: java.io.PrintStream err>
org.jfree.data.jdbc.JDBCXYDataset.close : $r4 = <java.lang.System: java.io.PrintStream err> >>>>>>>> org.jfree.data.jdbc.JDBCXYDataset.close : virtualinvoke $r4.<java.io.PrintStream: void println(java.lang.String)>("JdbcXYDataset: swallowing exception.")
org.jfree.data.jdbc.JDBCXYDataset.close : virtualinvoke $r4.<java.io.PrintStream: void println(java.lang.String)>("JdbcXYDataset: swallowing exception.") >>>>>>>> org.jfree.data.jdbc.JDBCXYDataset.close : return
org.jfree.data.jdbc.JDBCXYDataset.close : $r2 = r0.<org.jfree.data.jdbc.JDBCXYDataset: java.sql.Connection connection> >>>>>>>> org.jfree.data.jdbc.JDBCXYDataset.close : interfaceinvoke $r2.<java.sql.Connection: void close()>()
org.jfree.data.jdbc.JDBCXYDataset.close : $r2 = r0.<org.jfree.data.jdbc.JDBCXYDataset: java.sql.Connection connection> >>>>>>>> org.jfree.data.jdbc.JDBCXYDataset.close : $r3 := @caughtexception
org.jfree.data.jdbc.JDBCXYDataset.close : interfaceinvoke $r2.<java.sql.Connection: void close()>() >>>>>>>> org.jfree.data.jdbc.JDBCXYDataset.close : goto [?= return]
org.jfree.data.jdbc.JDBCXYDataset.close : interfaceinvoke $r2.<java.sql.Connection: void close()>() >>>>>>>> org.jfree.data.jdbc.JDBCXYDataset.close : $r3 := @caughtexception
org.jfree.data.jdbc.JDBCXYDataset.close : goto [?= return] >>>>>>>> org.jfree.data.jdbc.JDBCXYDataset.close : return
